runtime/doc/options.txt

```diff
@@ -2253,7 +2253,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	security reasons.
 
 					*'display'* *'dy'*
-'display' 'dy'		string	(default "")
+'display' 'dy'		string	(default "lastline", Vi default: "")
```

Comment:
Since we still have `set ...&vi`, and the values differ, I think it's better to document it.